Cargo.toml

```diff
@@ -7,3 +7,6 @@ repository = "https://github.com/dragostis/pest"
 documentation = "http://dragostis.github.io/pest/pest/"
 keywords = ["pest", "parser", "peg", "grammar"]
 license = "MPL-2.0"
+
+[dependencies]
+clap = "2.8.0"
```

Comment:
The only issue I see here is that you probably won't be able to run `cargo install pest` to get the binary since the main function will be behind a feature, so you will probably have to provide the feature when installing.